backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

```diff
@@ -61,10 +61,21 @@ spec:
     ttl: 720h0m0s
 EOF
 ----
+
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
++
+[NOTE]
+====
+To schedule a backup at specific intervals, enter the `<duration in minutes>` in the following format:
+----
+  `schedule: "*/10 * * * *"`
+----
```

Comment:
Also you don't need the backticks within a code block, which should have `[source,terminal]` as well:
```suggestion
[source,terminal]
----
  schedule: "*/10 * * * *"
----
```